rt-shell/objects/obj_shell/Other_10.gml

```diff
@@ -14,6 +14,14 @@ variable_global_set("sh_clear", function(args) {
 		return "";
 	}
 });
+variable_global_set("hint_clear", function() {
```

Comment:
gonna need to update this to `meta_clear`

Reply:
Haha oops, fixed!